my_sprites.py

```diff
@@ -319,6 +319,9 @@ def __init__(self,
         # angle used for the dir the sprite is facing. for the enemy this will usually be the angle to the target
         self._direction = 0  # direction facing. Should be in degrees
 
+        # point which actually hits target during attack
+        self.attack_point = None
```

Comment:
This would need to go into the _Weapon_. And probably be a list (think of a hand grenade)